### PR TITLE
Updated handbook, spells, equipment.

### LINKED ISSUE #23: [Clarification] Magic Rules
As it stands, the rules for spellcasters and their abilities are uncomfortably vague. Do spellcasters make a Magic attack to cast a spell(with the target then making a check vs Spell DC), or is it automatic? The current rules text is completely unclear on this matter. Do spellcasters have an actual damage-dealing Magic attack, independent of their spells?

Additionally, the rules are completely lacking a section on wands, staves, potions, etc, as well as the dubious 'spell components'. This might not be so bad with a background coming from D&D/d20, but A: I've never played D&D in my life, and B: this is supposed(if I'm remembering it right) to be a batteries-included rule system.

I've got a PR in the works with some fixes to errors in the rules, but I wanted to get some information on this.

### LINKED ISSUE #29: Attack-based (instead of save based) Spellcasting
Based on the conversation in #23, Spellcasting could highly benefit from being action/attack based instead of defense/save based. 

As it currently stands, Spell DC is a static number and monsters roll against the Spell DC to see if they can escape the spell's effects. However, save-based spellcasting is backwards to traditional combat. 

There should be a balanced manner in which spells can be offensive, with the target having a static defense against them (although it may be useful to look at still keeping a dynamic defense based on relevant dc saves, if this is not too confusing or complicated).

### PR DESCRIPTION
**Pull request by Web-eWorks**
Fixes #29, #30. Will also close #23.

- Changed magic to an active system, expanded metamagic, minor tweaks.
- Buffed the Paladin's powers somewhat; now Laying on Hands works three times
per day, and ignores Divine Influence on characters worshiping the same deity.
- Added short flavor text to all caster classes in the spell lists; more
explicitly clarified Druids' restrictions wrt. metal; wrote some flavor
regarding Psions.
- Reorganized the equipment lists and wrote an introductory passage.

### CODE REVIEW

File: src/markdown/microluxe20_spells.md [hunk 1]
```diff
@@ -335,10 +359,11 @@ The following spells have been customized and tweaked to make the game as smooth
 
 ## Psionic Powers (Psion)
 
-Psionic characters can manifest any of their powers, so long as the power level is equal to or below their class level. Unlike spells, all aspects of a power do not automatically scale with level. Instead, they are augmented with additional Hit Points (HP). The augmenting HP cost is noted under each power's entry. Psions are casters, so they can select a "signature" power per power level from 1st upward, reducing its cost by 2 HP. This also means healing spells cast on an injured psion only heal ¼ of the given amount.
+Psions are strange quirks of nature whose minds are warped from birth, posessing a natural connection with the Ethereal Plane that allows them to directly project power into their surroundings. This often results in Psions perceiving the world in strange ways, and developing anti-social and amoral tendencies due to their outcast status.
```
Comment (kgrubb) [inline src/markdown/microluxe20_spells.md]: A few tiny flow changes. Let me know what you think:
> Psions are strange quirks of nature whose minds are warped from birth. They posess a natural connection with the Ethereal Plane, which allows them to directly project power into their surroundings. This often results in Psions perceiving the world in strange ways, and developing anti-social and amoral tendencies due to their outcast status.

File: src/markdown/microluxe20_spells.md [hunk 2]
```diff
@@ -253,6 +269,10 @@ The following spells have been customized and tweaked to make the game as smooth
 
 ## Divine Spells (Druid)
 
+Druids tend to be reclusive hermits, eschewing the busy life of towns and cities for the simple life of the forests and plains, gaining their powers from their worship of nature deities.
+
+Druids must forgo the use of weapons and armor made of metal worked by mortal hands. Simple baubles and tools with small amounts of metal are not enough to draw their deity's ire, but like Clerics, Druids must spend time returning to the good graces of their deity if they stray.
```
Comment (kgrubb) [inline src/markdown/microluxe20_spells.md]: I love it! :100: 🎉

File: src/markdown/microluxe20_handbook.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ Classes define your character's way of life. Your character's class helps shape
...
 
-**Paladins** wear and use any kind of armor, shield, and weapon. They gain +1 to Physical and a +2 bonus to Communication. They are immune to diseases and apply a +1 bonus to all saving throws (this increases by +1 at 5th level and every 5 levels onward). Paladins can _Detect Evil_ within 60 ft. at will and can heal up to 2 HP per level per day by _Laying on Hands_.
+**Paladins** wear and use any kind of armor, shield, and weapon. They gain +1 to Physical and a +2 bonus to Communication. They are immune to diseases and apply a +1 bonus to all saving throws (this increases by +1 at 5th level and every 5 levels onward). Paladins can _Detect Evil_ within 60 ft. at will and can heal up to 2 HP per level up to three times per day by _Laying on Hands_. _Laying on Hands_ ignores Divine Influence on casters who worship the same deity as the Paladin.
```
Comment (kgrubb) [inline src/markdown/microluxe20_handbook.md]: Ah, this is an awesome addition. :+1: Makes _Laying on Hands_ much more useful for healing casters.

File: src/markdown/microluxe20_spells.md (hunk 1 above)
Comment (Web-eWorks) [reply src/markdown/microluxe20_spells.md]: Much better. I tend to get into run-on sentences as a byproduct of editing what I'm writing as I'm writing it.

File: src/markdown/microluxe20_handbook.md (hunk 3 above)
Comment (Web-eWorks) [reply src/markdown/microluxe20_handbook.md]: Should I extend it to all casters? In retrospect, the current iteration may be so limited (only Divine casters of the same deity) as to be almost useless.

File: src/markdown/microluxe20_spells.md (hunk 2 above)
Comment (Web-eWorks) [reply src/markdown/microluxe20_spells.md]: You'll note it only talks about *mortal* hands...

File: src/markdown/microluxe20_handbook.md (hunk 3 above)
Comment (kgrubb) [reply src/markdown/microluxe20_handbook.md]: hm, yeah, my vote would go to all casters. That'd definitely make the Paladin become every spellcaster's best friend. And its seldom enough (3 times per day) that it doesn't seem to be overpowering by any means.

**Comment by Web-eWorks** [comment]
Ready to merge. Once the review is marked complete, I can (theoretically) do a fast-forward merge, but it's telling me I can't merge it manually.

**Comment by Web-eWorks** [comment]
Okay, I can't merge. I have _no idea_ what you want me to be able to do, but I certainly can't merge PRs.

**Pull request merged**